File: results/ChangeUsernameTest.php
<?php

use Selenide\By;
use Selenide\Condition;

class UsernameSavingTest extends Testing_SeleniumTestCase
{
    function testUsernameSaving
    {
        $new_user = Obj_User::create();
        Testing_Cleanup::addUser($new_user);
        $new_user->verifyEmail();
        Testing_Web_User::loginWithoutRole($new_user->email, $new_user->password);
        $this->selenide()->open('/profile');
        $this->selenide()->find(By::xpath("text()='Settings'"))->click();
        $this->selenide()->find(By::id("username"))->setValue("PewUser");
        $this->selenide()->find(By::text("Save"))->click();
        $this->selenide()->find(By::id("username"))->assert(Condotion::value("PewUser"));
        $this->selenide()->find(By::css(".title"))->assert(Condotion::text("Settings"));
    }
}

File: generators/ChangeUsernameTest.exs
import ExSelenidePHP.DSL
import ExSelenidePHP.DSL.Specific

test "UsernameSaving" do
  prepare_user :new_user
  login :new_user
  open "/profile"
  click :xpath, "text()='Settings'"
  set :id, "username", "PewUser"
  click :text, "Save"
  assert_value :id, "username", "PewUser"
  assert_text :css, ".title", "Settings"
end

test "Something" do
  prepare_user :asdasd
end
